testutils/resourcebuilder/machine/v1beta1/gcp_provider_spec.go

```diff
@@ -120,3 +122,9 @@ func (m GCPProviderSpecBuilder) WithZone(zone string) GCPProviderSpecBuilder {
 	m.zone = zone
 	return m
 }
+
+// WithDisks sets the disks for a machine config builder.
+func (m GCPProviderSpecBuilder) WithDisks(disks []*machinev1beta1.GCPDisk) GCPProviderSpecBuilder {
+	m.disks = disks
+	return m
+}
```

Comment:
I know Joel likes these to maintain alphabetical ordering. Same goes for the default setting above.